shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/future/lock/service/GlobalLockRegistryService.java

```diff
@@ -40,7 +58,7 @@ public GlobalLockRegistryService(final ClusterPersistRepository repository) {
      * @return true if get the lock, false if not
      */
     public boolean tryLock(final String lockName, final long timeoutMilliseconds) {
-        return repository.tryLock(lockName, timeoutMilliseconds, TimeUnit.MILLISECONDS);
+        return repository.persistEphemeral(lockName, LockAck.LOCKED.name());
```

Comment:
Just catch exception here, it is better to keep original return type with `persistEphemeral`

Reply:
OK, I will fix them.